### PR TITLE
Add omi.keytab update to the installer

### PR DESCRIPTION
**Pull request by yakman2020**
The scripts are placed on all systems as we probably can support these systems using kerberos. We just haven't tested it. 

The cronjob to update is only started/stopped on Linux.

### CODE REVIEW

**Comment by yakman2020** [comment]
@Microsoft/omi-devs

File: Unix/tools/ktstrip [hunk 1]
```diff
@@ -43,59 +43,28 @@ then
...
-       host/*) printf  $principalname"\n" ;;
-       RestrictedKrbHost/*) printf $principalname"\n" ;;
-       *) printf "remove " $principalname "\n" 
+printf "read_kt $SRC_KT\nlist\n" | ktutil >$list  
 
-          remove_entries+=($entryno)
-          ;;
-    esac
-done
+# sort -rV reverse sorts the entries by "version number" which is to say actual numeric value. -n sorts by digit.
```
Comment (jeffaco) [inline Unix/tools/ktstrip]: Would be really nice to have sample output and a description (in comment) of exactly what you're doing here.

File: Unix/tools/ktstrip [hunk 2]
```diff
@@ -43,59 +43,28 @@ then
...
-printf "write_kt /tmp/t.kt\n" >>$TMP_FILE
-
-cat $TMP_FILE | ktutil
-mv /tmp/t.kt $DST_KT
+printf "Creating $DST_KT\n"
+cat $cmd | ktutil
+printf "Move $tmp_kt to $DST_KT\n"
+mv $tmp_kt $DST_KT
 
 chown $user $DST_KT
```
Comment (jeffaco) [inline Unix/tools/ktstrip]: Note that `$user` may not be passed in. Personally, I prefer it be hardcoded.

File: Unix/tools/ktstrip [hunk 3]
```diff
@@ -7,23 +7,23 @@ user=$3
 
 SRC_KT=$(readlink -f $1)
```
[the remark this answers is not in the document]
Comment (yakman2020) [reply Unix/tools/ktstrip]: scripts are restricted to linux. Didn't see any problems running them on AIX or Solaris, but hpux hasn't been tried.

Added comments in ktstrip describing in deatil whats going on.

File: Unix/installbuilder/datafiles/Base_OMI.data [hunk 4]
```diff
@@ -40,11 +42,13 @@ PAM_CONF_DIR: '/etc/pam.d'
 /opt;                                    755; root; ${{ROOT_GROUP_NAME}}; sysdir
 /opt/omi;                                755; root; ${{ROOT_GROUP_NAME}}
 /opt/omi/bin;                            755; root; ${{ROOT_GROUP_NAME}}
+/opt/omi/bin/support;                    755; root; ${{ROOT_GROUP_NAME}}
 /etc/opt;                                755; root; ${{ROOT_GROUP_NAME}}; sysdir
 /etc/opt/omi;                            755; root; ${{ROOT_GROUP_NAME}}
 /etc/opt/omi/conf;                       755; root; ${{ROOT_GROUP_NAME}}
 /etc/opt/omi/conf/omiregister/root-omi;  755; root; ${{ROOT_GROUP_NAME}}
 /etc/opt/omi/ssl;                        755; root; ${{ROOT_GROUP_NAME}}
+/etc/opt/omi/creds;                      755; root; ${{ROOT_GROUP_NAME}}
```
[the remark this answers is not in the document]
Comment (yakman2020) [reply Unix/installbuilder/datafiles/Base_OMI.data]: 700 or 750 on the cred dir results in not being able to build a package

File: Unix/tools/ktstrip (hunk 3 above)
[the remark this answers is not in the document]
Comment (yakman2020) [reply Unix/tools/ktstrip]: readlink is in AIX and solaris. Note from solaris 8 :
http://docs.oracle.com/cd/E19109-01/tsolaris8/816-1056/6m7gi1i3r/index.html. Its not in hpux. Clearly we won't support hpux. That also takes care of -nt.

**Pull request merged**